src/OrchardCore/OrchardCore.Logging.NLog/WebHostBuilderExtensions.cs

```diff
@@ -9,6 +10,7 @@
 
 namespace OrchardCore.Logging
 {
+    [Obsolete("Use HostBuilderExtensions instead.")]
```

Comment:
Can we use HostBuilderExtensions without changing all the calls like `.Services`? If not then I would not mark it otherwise all users upgrading to this will see the obsolete warning.

Reply:
Removed `Obsolete` as `IWebHostBuilder` and/or `IHostBuilder`  can be used to configure `NLog`